src/components/Attachments/AttachmentView/index.js

```diff
@@ -132,6 +133,7 @@ function AttachmentView({
                 onScaleChanged={onScaleChanged}
                 onToggleKeyboard={onToggleKeyboard}
                 onLoadComplete={() => !loadComplete && setLoadComplete(true)}
+                errorLabelStyles={isUsedInAttachmentModal ? [styles.textLabel, styles.textLarge] : []}
```

Comment:
Shouldn't we add `cursor.cursorAuto` only if `isUsedInAttachmentModal=true`?

Reply:
Yes we could, I was thinking maybe we can omit the whole Text change thing, and stick with only the small size text for both cases. that will remove so much overhead. what do you think. ?

Reply:
I don't think it's that much overhead, ideally, we should retain the current design